src/styles/blocks/main-title.scss

```diff
@@ -0,0 +1,218 @@
+.main {
+
+  &__container {
+    padding: 0 20px;
+    margin-bottom: 60px;
+
+    @include onTablet {
+      padding: 0 35px;
+      margin-bottom: 100px;
+    }
+
+    @include onDesktop {
+      padding: 0 55px;
+      margin-bottom: 100px;
+    }
+
+    &--nevs {
+      display: block;
+      @include onDesktop {
+        display: grid;
+      }
+    }
+
+    &--date {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+
+      @include onTablet {
+        justify-content: flex-start;
+      }
+
+      @include onDesktop {
+        justify-content: flex-start;
+      }
+    }
+  }
+
+  &__exhib {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 16px;
+    color: #687480;
+
+    @include onTablet {
+      font-size: 14px;
+      line-height: 14px;
+      margin-right: 30px;
+    }
+
+    @include onDesktop {
+      font-size: 14px;
+      line-height: 14px;
+      margin-right: 30px;
+    }
+  }
+
+  &__date {
+    font-family: "IBM Plex Sans", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 24px;
+    color: $green;
+
+    @include onTablet {
+      line-height: 21px;
+    }
+
+    @include onDesktop {
+      line-height: 21px;
+    }
+  }
+
+  &__subtitle {
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 28px;
+    line-height: 28px;
+    color: #0f0e08;
+    margin: 0 0 10px;
+
+    @include onTablet {
+      margin: 0 0 15px;
+    }
+
+    @include onDesktop {
+      margin: 0 0 15px;
+    }
+
+    &::after {
+      content: "";
+      width: 10px;
+      height: 10px;
+      background-color: #f14a27;
```

Comment:
Consider using a variable for the color #f14a27 to maintain consistency and ease of updates.